### PR TITLE
Cleanup lform directory and bug fixes

### LINKED ISSUE #6089: Encounter form delete shows wrong form name questionnaires
### Describe the bug
Questionnaires work flow for form selections are different than normal encounter forms thus a custom hook is needed.
Not really a bug that prevents correct delete of form but the confirm verbiage is incorrect causing confusion.

### LINKED ISSUE #6092: CCM author or other setting missing date create
### Describe the bug
CCM is sending a create date for setting items.
Unknown why but I think is needed for ONC.
Fixing in master and v7 release
### Client configuration

- Browser:
- OpenEMR version: v7(2) and master
- Operating system:

### PR DESCRIPTION
**Pull request by sjpadgett**
- Remove un-needed files left over by previous version of LHC-Forms

Fixes #6089
Fixes #6092

### CODE REVIEW

File: src/OeUI/OemrUI.php [hunk 1]
```diff
@@ -499,6 +499,8 @@ private function headerActionJs($arrAction = array())
         $action_top_js = <<<SHWTOP
         <script>
         $(function () {
+            let element = document.querySelector('a#show_hide i');
+            let elementIcon = document.querySelector('a#show_hide i');
```
Comment (sjpadgett) [inline src/OeUI/OemrUI.php:503]: @robertdown 
Depending on the initial environment state will determine if this is needed or not.
I think once local storage is set but I stopped trying to follow the whole OemrUI logic.
Declaring the var to be more global for where it is needed is the fix.

**Comment by sjpadgett** [comment]
I seem to have forgotten why I started this PR!:)

**Comment by bradymiller** [comment]
2DV

File: src/OeUI/OemrUI.php (hunk 1 above)
Comment (stephenwaite) [reply src/OeUI/OemrUI.php:503]: CzLY2Xbq

**Pull request merged**